packages/calcite-components/src/components/tab-title/tab-title.scss

```diff
@@ -1,3 +1,25 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-tab-title-background-color: Specifies the background color of the component.
+ * @prop --calcite-tab-title-background-color-hover: Specifies the background color of the component when hovered.
+ * @prop --calcite-tab-title-border-color-selected: Specifies the border color of the component. Only applicable when bordered.
```

Comment:
I think this one is a little confusing. I'd interpret this to mean the "accent" or "highlight" color. But it seems like it would just be the two "side borders"? Maybe this can be set by the tab nav if that is the case.

I'm working on #8800 which would make this useful - currently the selection element is not part of Tab Title, but it will when the above fix lands.

Reply:
> I think this one is a little confusing. I'd interpret this to mean the "accent" or "highlight" color. But it seems like it would just be the two "side borders"? Maybe this can be set by the tab nav if that is the case.

Agreed. Let me see if I can move this up to tab nav and simplify the props here.

> I'm working on #8800 which would make this useful - currently the selection element is not part of Tab Title, but it will when the above fix lands.

Do you estimate this landing in this release? I could proceed with the current props/styles and revisit once the selection indicator is installed.

Reply:
I think you could still add the highlight css prop and in my PR I can change the element it affects from the "indicator div" to just the border, and keep the name the same.